examples/angular/src/app/pages/components/footer/destinations/destinations.component.ts

```diff
@@ -1,16 +1,15 @@
 import {
   Component,
-  InjectionToken,
   OnInit,
   inject,
   signal,
 } from '@angular/core';
 
-import { GenericContentlet } from '..';
-import { ContentletsComponent } from '../contentlets/contentlets.component';
+import { GenericContentlet } from '../..';
+import { ContentletsComponent } from '../../../../shared/contentlets/contentlets.component';
```

Comment:
Can we create absolute `@paths`?

Reply:
I think is not neccesary for the example